cypress/integration/home.js

```diff
@@ -1,12 +1,20 @@
 describe('My First Test', () => {
```

Comment:
Let's name this over the page we're testing
```suggestion
describe('Home Page', () => {
```